.github/workflows/ci.yml

```diff
@@ -88,82 +95,10 @@ jobs:
           npm ci
           gulp
 
+      - name: Run phpstan
+        run: ddev php ./vendor/bin/grumphp run --tasks=phpstan
+
       - name: Test
         run: |
           ddev phpunit --testsuite unit
           ddev phpunit --bootstrap=./vendor/weitzman/drupal-test-traits/src/bootstrap-fast.php --configuration ./phpunit.xml --testsuite existing-site
-
-
-  frontend_vr:
```

Comment:
Why are you removing this block? We need to fix the failures here. Right now, it doesn't block deployment anyway.

Reply:
No Its not removed. Its still in WIP. I didn't reach there yet. I started working taking job by job. I will take it after finalising `drupal_test`.